src/export/ExportWavPack.cpp

```diff
@@ -95,27 +95,31 @@ const std::vector< int > ExportQualityValues{
    3,
 };
 
-namespace {
 
-const TranslatableStrings ExportBitDepthNames{
-   XO("16 bit") ,
-   XO("24 bit") ,
-   XO("32 bit float ") ,
-};
-
-const std::vector< int > ExportBitDepthValues{
-   16,
-   24,
-   32,
-};
+extern IntSetting QualitySetting;
```

Comment:
This commit reminds me that I turn off wavpack in my local build because for some reason I never got it to work.  `wavpack/wavpack.h` can't be found.

Reply:
If I choose `system` not `local` for `audacity_use_wavpack` it doesn't even configure.

Reply:
I'm not sure why it should. Do you have a system version of wavpack installed?

>This commit reminds me that I turn off wavpack in my local build because for some reason I never got it to work. wavpack/wavpack.h can't be found.

I was sure that we have overcame this problem by dropping your Conan cache.